site/docs/references/formatters.md

```diff
@@ -55,7 +55,9 @@ For example a `font-size` formatter can return a different value for the font si
 
 ## Formatters with variables
 
-You can use formatters with variables:
+You can use formatters with variables. 
+
+In this example the CSS imports the same formatter as the previous example, `cal-font-size`, but the variable `baseFontSize` is added to the calculation.  
 
 ```css
     :import {
```

Comment:
in the var definition - remove the px, as it will be added from the formatter (otherwise we'll get something like `12pxpx`)